desktop-src/direct3dhlsl/QuadAny.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Compares boolean across a quad.
+nms.assetid:
+title: QuadAny
+ms.topic: reference
+ms.date: 07/10/2024
+topic_type:
+- APIRef
+- kbSyntax
+api_name:
+- QuadAny
+api_type:
+- NA
+---
+
+
+# QuadAny
+
+Compares boolean across a quad.
+
+## Syntax
+
+
+```syntax
+bool QuadAny(bool cond);
+```
+
+## Parameters
+
+| Item | Description |
+|------|-------------|
+| *cond* | [in] A vector or scalar variable of type boolean which we want to evaluate. QuadAny will return true if any component of this variable is true.  |
+## Return value
+
+ Returns a scalar of type boolean, true if any component of the parameter is true, otherwise it returns false.
```

Comment:
Interesting that it recognized this is about the quad for `all` but not for `any`